public/_css/main.css

```diff
@@ -1 +1 @@
e}pre{white-space:pre;white-space:pre-wrap}table #results-1{width:100px}table pre,table td.code{width:650px}mark{background:#ff9;padding:.2em .1em}h1,h2,h3,h4{font-size:1em}h1{padding-top:1em;font-size:1.4em}form h3{padding-top:.2em;padding-bottom:.2em}h1 strong{font-style:italic;font-family:Monaco,'Lucida Console',monospace}li.current a{background:#90ee85}footer{margin-top:2em;border-top:2px solid #c4c4c4;font-size:.9em;overflow:hidden}footer ul{list-style:none;padding:0;margin:0}footer>ul>li{float:left;width:15%}footer>ul>li:first-child{width:55%}footer ul ul li{margin-bottom:.1em}footer h4{margin-bottom:.2em}#add-test{margin-right:.3em}#bs-chart-frame{height:240px;width:100%}#bs-logo{margin:0}#bs-logo span,applet{position:absolute;left:-9999em}#bs-logo a{width:232px;height:39px;filter:none;background:url(//www.browserscope.org/static/img/logo.png) no-repeat}#bs-ua{padding:.5em .5em 0;color:#555}#bs-results .bs-rt{font-size:10pt;padding:.5em;background:#ddd}#bs-results td{border:1px solid #ddd;padding:.4em;white-space:nowrap}#bs-results .rt-ua-cur{font-style:italic}#bs-results .bs-rt-message{padding:3em;text-align:center;font-weight:700;color:#555}#bs-results .google-visualization-table-tr-head td{white-space:normal}#controls{margin-top:-3.35em}#comments h1{padding:0}#comments .meta img{position:absolute;margin:0;top:2px;left:2px}#comments .meta{padding-left:35px;margin-top:0;width:923px;line-height:30px}#comments .meta a{color:#555}#comments article div{padding:0 1em}#comments article{border:1px solid #b4b4b4;position:relative;margin-bottom:1em}#comments article:target{background:#dde4ea}#error-info.show,#firebug strong,#java strong,#status strong,.meta strong{background:pink;border:1px solid #b00b00;padding:.4em}#error-info.show{padding:.5em 1em}#error-info,#slug,code,samp,textarea,var{font-family:Monaco,monospace;font-size:.9em;-moz-tab-size:2;-o-tab-size:2;tab-size:2}#java strong{background:#ffffdc;border:1px solid #faa700}#slug{font-size:1em}#faq dt{margin-top:1em}#faq dt:hover a{display:inline}#faq dt:target,#faq dt:target+dd{background:#90ee85;margin:0 -.8em;padding:0 .8em}#faq dt:target+dd{padding-bottom:.5em;margin-bottom:-.5em}#faq dt:target{margin-top:.5em;padding-top:.5em}#firebug,#java,#status{margin:0 0 1em;padding:.3em 0}#prep-code pre{max-height:500px;overflow:auto}.co1,.co2,.coMULTI{font-style:italic;color:#060}.error{color:#b00b00}.imp{color:red}.kw1,.kw3{color:#006}.kw2{color:#036}.es0{color:#009}.br0{color:#090}.sy0{color:#393}.st0{color:#36c}.nu0{color:#c00}.me1{color:#606}#carbonads{float:right;width:130px;font-size:80%}@media (max-width:1050px){table pre,table td.code{width:550px}}@media (max-width:1040px){body{margin:0;border:0}#comments .meta,body{width:100%;-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}}@media (max-width:800px){table pre,table td.code{width:450px}}@media (max-width:680px){table pre,table td.code{width:350px}}@media (max-width:650px){table pre,table td.code{width:200px}}
```

Comment:
I didn't make intentional changes to the CSS for this PR. Diff is a consequence of `npm run build` which I ran to make updated `test.js`